File: Base_TechFlow/src/main/java/com/techflow/ws/sys/entity/SysUser.java
package com.techflow.ws.sys.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

/*
 * Usuários do Sistema
 */
@Entity
@Table(name="tflow_usuario")
public class SysUser extends CommonTimestampEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="idusuario", length=9, nullable=false)
    private Integer id;

    @Column(name="usuario", length=50, nullable=false)
    private String username;

    /*
     * Senha criptografada
     */
    @Column(name="senha", length=100, nullable=false)
    private String password;

    @Column(name="nome", length=100, nullable=false)
    private String name;

    /*
     * Email do Usuário
     */
    @Column(name="email", length=100, nullable=false)
    private String email;

    /*
     * Session ID para controle de sessão
     */
    @Column(name="sid", length=100, nullable=true)
    private String sid;

    public SysUser() {
    }

    public SysUser(Integer id) {
        this.id = id;
    }

    public SysUser(String username, String password, String name, String email) {
        this.username = username;
        this.password = password;
        this.name = name;
        this.email = email;
    }

    public Integer getId() {
        return id;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }

    public String getEmail() {
        return email;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getSid() {
        return sid;
    }

    public void setSid(String sid) {
        this.sid = sid;
    }
}

File: Techflow_Base/Techflow_Windows_Fonte/SysUser.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Techflow
{
    public class SysUser
    {
        private int _userId;
        private string _username;
        private string _name;
        private string _email;
        private string _password;

        public int UserId
        {
            get { return _userId; }
            set { _userId = value; }
        }
        public string Username
        {
            get { return _username; }
            set { _username = value; }
        }
        public string Email{
            get { return _email; }
            set { _email = value; }
        }
        public string Name
        {
            get { return _name; }
            set { _name = value; }
        }

        public string Password
        {
            get { return _password; }
            set { _password = value; }
        }
    }
}
